core/ArchiveProcessor/Loader.php

```diff
@@ -117,8 +118,7 @@ private function prepareArchiveImpl($pluginName)
         // with a ts_archived >= the DONE_OK/DONE_INVALIDATED date.
         list($idArchives, $visits, $visitsConverted, $isAnyArchiveExists) = $this->loadExistingArchiveIdFromDb();
         if (!empty($idArchives)
-            && !$this->params->getArchiveOnlyReport()
```

Comment:
Removed this since it seemed safer to reuse existing archives if we're not forcing single plugin archiving, and not depend on this state which comes from a query param.